app/src/main/java/com/timothyolt/evolutionarydesign/auth/AuthenticationActivity.kt

```diff
@@ -58,7 +66,7 @@ class AuthenticationActivity : AppCompatActivity() {
         } else null
     }
 
-    data class Authentication(
+    private data class Authentication(
```

Comment:
Probably don't want this to be private since we'll need it's info to upload images. Might even want to make it serializable so that we can write it to sharedprefs or something.

Reply:
yeah good thought, I'm going to push that until when we need it though